src/http/mod.rs

```diff
@@ -1,3 +1,5 @@
+pub mod kv;
```

Comment:
it felt appropriate to namespace the http mods based on the services we use, i.e. workers and kv, so you might call, in this case, `kv::create_namespace("title")`.